services/user.py

```diff
@@ -0,0 +1,42 @@
+from db.models import User
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: str = None,
+    first_name: str = None,
+    last_name: str = None,
+) -> User:
+
+    user = User.objects.create_user(
+        username=username,
+        password=password,
+    )
+
+    if email:
+        user.email = email
+
+    if first_name:
+
+        user.first_name = first_name
+
+    if last_name:
+        user.last_name = last_name
+
+    user.save()
+    return user
+
+
+def get_user(user_id: int) -> User:
+    return User.objects.get(id=user_id)
+
+
+def update_user(user_id: int, password: str = None, **kwargs) -> User:

```

Comment:
Consider handling the potential `DoesNotExist` exception when retrieving the `User` instance in `update_user`. This will prevent the application from crashing if the user ID does not exist.